course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,257 @@
+    * [Lecture topics](#lecture-topics)
+    * [Discussions](#discussions)
+    * [Exercises](#exercises)
+
+<!-- vim-markdown-toc -->
+
+## On the necessary changes to the RQM syllabus
+- To tailor the course for ecology students, we need to remove some of the existing material.
+- I have tried to summarize the differences between the original schedule and the schedule I propose further down. I did this by classifying all the lectures and tutorials into the overarching themes of the RQM course, and quantified the number of classes dedicated to a theme in the original and proposed schedules (original -> proposed). This can serve as a rough overview for how the course direction is changing:
+    - 13 -> 8.5 Reproducibility (includes all R classes)
+    - 5 -> 1.5 Authorship, collaboration and publishing
+    - 6 -> 0 Openness in data and science
+    - 5 -> 8 Project work
+    - 0 -> 6 Theoretical ecology
+- Most of the topics in Christie's course involve both practical and theoretical aspects. For example, all of the R-coding would fall under reproducibility, but there are also discussions dedicated to reproducibility.
+- Note that there are 2 or 3 of these topics per week in the original RQM schedule. Time-wise, this means that a topic approximately equals one class (lecture or tutorial). There were 29 topics in the original RQM course, which we have reduced to 24 topics to fit our course. The total lecture time is still the same, just that each topic gets more attention.
+
+## Draft of proposed schedule
+- An immediate conflict to solve is how to ensure that the students get started with their project early, but also have the necessary knowledge about data analyses, R, and the ecology models that Martin wants to incorporate.
+- As we are now incorporating some ecology classes into this schedule, I think it is important to make sure enough time is dedicated to teaching sound principles around the data analyse pipeline, as this is still the main focus of the course in my opinion.
+- We will teach the introduction with a common data, since this will make it faster, clearer and force students to go through these procedures at least twice, once with the common data set and once with their own, which is advantageous or knowledge retention.
+- I originally split these topics into lecture and tutorial, but I will just call it 1 and 2 instead, since both of them will hopefully be in a computer lab. The third item under each week is for suggested homework.
+- Quizzes will be done using the automated blackboard setup to as large extent as possible. This will likely work very well for the initial programming multiple choice type exercises, together with some coding puzzles that would be manually graded.
+    - This will give students a fair estimate on where they stand before the course drop rate.
+- If possible we would like to record the classes, at least the slides and speaker voice through Google Hangouts as during our UofTCoders event.
+
+### Week 1
```

Comment:
We should soon discuss who will be teaching what. I already have some ideas for this, but they don't line up perfectly with the schedule, and maybe people should voice their interest before I go about proposing classes.

Will everyone be present for the first class? Currently the classes line up with the proposed teaching schedule of 2 weeks in a row per person, but we change this also if needed.